### PR TITLE
Login Success Message

### PR DESCRIPTION
**Pull request by mdelez**
closes https://dasch.myjetbrains.com/youtrack/issue/DSP-331

### CODE REVIEW

**Comment by mdelez** [comment]
@flavens do you have an opinion on what we should do about the default message CSS having a background color opacity of 0.4? This leads to text on the page overlapping with the message box. Should we remove the opacity or make it so that the message box doesn't cover any text on the page? We could also make it so that the width of the message box is equivalent to the length of the message text (with a few pixels of padding) which would solve this issue for this case but maybe not on other pages where there is text in the middle of the page.

Screen Shot 2020-09-16 at 10 44 45

**Comment by mdelez** [comment]
change to the css will be done in https://github.com/dasch-swiss/dsp-ui-lib/pull/180

File: src/app/main/header/header.component.ts [hunk 1]
```diff
@@ -45,6 +57,11 @@ export class HeaderComponent {
         });
     }
 
+    ngOnInit() {
+        this.componentCommsSubscription = this._componentCommsService.on(
```
Comment (flavens) [inline src/app/main/header/header.component.ts:61]: you should unsubscribe to the subscription

File: src/app/main/header/header.component.spec.ts [hunk 2]
```diff
@@ -94,4 +100,11 @@ describe('HeaderComponent', () => {
         expect(searchPanel).toBeDefined();
     });
 
+    it('should display the login success message when the LoginSuccess event is emitted', () => {
```
Comment (flavens) [inline src/app/main/header/header.component.spec.ts:103]: you can test the unsubscription

**Review by flavens — CHANGES_REQUESTED** [review]

File: src/app/main/header/header.component.ts (hunk 1 above)
Comment (mdelez) [reply src/app/main/header/header.component.ts:61]: good catch! fixed in 73b3db3

File: src/app/main/header/header.component.spec.ts (hunk 2 above)
Comment (mdelez) [reply src/app/main/header/header.component.spec.ts:103]: fixed in 73b3db3

**Review by flavens — APPROVED** [review]

**Pull request merged**